pkg/tnf/handlers/podsets/podsets.go

```diff
@@ -31,80 +31,83 @@ const (
 )
 
 // Deployment holds information about a single deployment
-type Deployment struct {
+type PodSet struct {
 	Replicas    int
 	Ready       int
 	UpToDate    int
 	Available   int
 	Unavailable int
+	Current     int
 }
 
 // DeploymentMap name to Deployment
-type DeploymentMap map[string]Deployment
+type PodSetMap map[string]PodSet
 
 // Deployments holds information derived from running "oc -n <namespace> get deployments" on the command line.
-type Deployments struct {
-	deployments DeploymentMap
-	namespace   string
-	result      int
-	timeout     time.Duration
-	args        []string
+type PodSets struct {
+	podsets   PodSetMap
+	namespace string
+	result    int
+	timeout   time.Duration
+	args      []string
 }
 
 // NewDeployments creates a new Deployments tnf.Test.
-func NewDeployments(timeout time.Duration, namespace string) *Deployments {
-	return &Deployments{
+func NewPodSets(timeout time.Duration, namespace, resourceType string) *PodSets {
+	return &PodSets{
 		timeout:   timeout,
 		namespace: namespace,
 		result:    tnf.ERROR,
-		args: []string{"oc", "-n", namespace, "get", "deployments", "-o", "custom-columns=" +
+		args: []string{"oc", "-n", namespace, "get", resourceType, "-o", "custom-columns=" +
 			"NAME:.metadata.name," +
 			"REPLICAS:.spec.replicas," +
 			"READY:.status.readyReplicas," +
 			"UPDATED:.status.updatedReplicas," +
 			"AVAILABLE:.status.availableReplicas," +
-			"UNAVAILABLE:.status.unavailableReplicas",
+			"UNAVAILABLE:.status.unavailableReplicas," +
+			"CURRENT:.status.currentReplicas",
```

Comment:
After reading a bit about stateful sets, I see we don't have the currentReplicas value in the deployment output. If we want to use the same output, we should not record it, otherwise we will always get "<none>" for deployments, which could cause errors later in GetPodSets()